lib/perf_hooks.js

```diff
@@ -604,12 +604,12 @@ function monitorEventLoopDelay(options = {}) {
   if (typeof options !== 'object' || options === null) {
     throw new ERR_INVALID_ARG_TYPE('options', 'Object', options);
   }
-  const { resolution = 10 } = options;
+  const { resolution = 0 } = options;
```

Comment:
@Flarna @indutny ... this is changing the default is it not?

Reply:
Yes, since https://github.com/nodejs/node/pull/32018/commits/4e8a3421cb1b5d8782c7ec59220df96e1ab9e22c behavior has changed. Before `monitorEventLoopDelay()` used a 10ms timer, now it uses the precise variant

Reply:
ah, my mistake, I misread the conversation thread :-)